hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -1883,4 +1883,22 @@
 	 * @see org.hibernate.query.ImmutableEntityUpdateQueryHandlingMode
 	 */
 	String IMMUTABLE_ENTITY_UPDATE_QUERY_HANDLING_MODE = "hibernate.query.immutable_entity_update_query_handling_mode";
+
+	/**
+	 * By default, the IN clause expends to include all bind parameter values.
```

Comment:
Was curious about the feature so I read the patch: s/expends/expands/

Reply:
Thanks for the tip. The IN clause expends quite some resources too :D